### ISSUE TITLE
Webdriver for Chrome does not work in Github Actions on latest Ubunty

### ISSUE THREAD

**Issue by ivanovmp** [SergeyPirogov/webdriver_manager#526]
I set up a dummy repository (link [here](https://github.com/ivanovmp/Webdriver-Manager-Testing)) which only contains a sample script [`sample.py`](https://github.com/ivanovmp/Webdriver-Manager-Testing/blob/main/sample.py) and a [workflow](https://github.com/ivanovmp/Webdriver-Manager-Testing/blob/main/.github/workflows/main.yml) which installs `selenium` and `webdriver-manager` and runs the sample script.

`.github/workflows/main.yml`:
```
name: Main CI

on:
  push:
    branches: [ "main" ]
  pull_request:
    branches: [ "main" ]

jobs:
  build:

    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v3
    - name: install
      run: pip install selenium webdriver-manager
    - name: run-sample
      run: python3 sample.py
```

`sample.py`:
```
# selenium 4
from selenium import webdriver
from selenium.webdriver.chrome.service import Service as ChromeService
from webdriver_manager.chrome import ChromeDriverManager

driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))
```

Note that this piece code worked fine four months ago. However, now the workflow [fails](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5350191600/jobs/9702467070):
```
[WDM] - Downloading:   0%|          | 0.00/7.0[6](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5350191600/jobs/9702467070#step:4:7)M [00:00<?, ?B/s]
[WDM] - Downloading: 100%|██████████| [7](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5350191600/jobs/9702467070#step:4:8).06M/7.06M [00:00<00:00, 173MB/s]
Traceback (most recent call last):
  File "/home/runner/work/Webdriver-Manager-Testing/Webdriver-Manager-Testing/sample.py", line 6, in <module>
    driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))
  File "/home/runner/.local/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py", line 49, in __init__
    super().__init__(
  File "/home/runner/.local/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py", line 54, in __init__
    super().__init__(
  File "/home/runner/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py", line 206, in __init__
    self.start_session(capabilities)
  File "/home/runner/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py", line 291, in start_session
    response = self.execute(Command.NEW_SESSION, caps)["value"]
  File "/home/runner/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py", line 346, in execute
    self.error_handler.check_response(response)
  File "/home/runner/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py", line 245, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.
  (unknown error: DevToolsActivePort file doesn't exist)
  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
Stacktrace:
#0 0x55ea6fae[8](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5350191600/jobs/9702467070#step:4:9)4e3 <unknown>
#1 0x55ea6f817c76 <unknown>
#2 0x55ea6f840d78 <unknown>
#3 0x55ea6f83d02[9](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5350191600/jobs/9702467070#step:4:10) <unknown>
#4 0x55ea6f87bccc <unknown>
#5 0x55ea6f87b47f <unknown>
#6 0x55ea6f872de3 <unknown>
#7 0x55ea6f8482dd <unknown>
#8 0x55ea6f84934e <unknown>
#9 0x55ea6faa83e4 <unknown>
#[10](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5350191600/jobs/9702467070#step:4:11) 0x55ea6faac3d7 <unknown>
#[11](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5350191600/jobs/9702467070#step:4:12) 0x55ea6fab6b20 <unknown>
#[12](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5350191600/jobs/9702467070#step:4:13) 0x55ea6faad023 <unknown>
#[13](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5350191600/jobs/9702467070#step:4:14) 0x55ea6fa7b1aa <unknown>
#[14](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5350191600/jobs/9702467070#step:4:15) 0x55ea6fad16b8 <unknown>
#[15](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5350191600/jobs/9702467070#step:4:16) 0x55ea6fad1847 <unknown>
#[16](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5350191600/jobs/9702467070#step:4:17) 0x55ea6fae1243 <unknown>
#[17](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5350191600/jobs/9702467070#step:4:18) 0x7f95a5157b[43](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5350191600/jobs/9702467070#step:4:44) <unknown>
```

**Comment by SergeyPirogov** [comment]
Hello, need to understand which version of chrome and the version of webdriver is downloaded

**Comment by ivanovmp** [comment]
```
~ % pip show webdriver-manager
  
Name: webdriver-manager
Version: 3.8.[6](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5358131043/jobs/9719976985#step:4:7)
Summary: Library provides the way to automatically manage drivers for different browsers
Home-page: https://github.com/SergeyPirogov/webdriver_manager
Author: Sergey Pirogov
Author-email: automationremarks@gmail.com
License: 
Location: /home/runner/.local/lib/python3.[10](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5358131043/jobs/9719976985#step:4:11)/site-packages
Requires: packaging, python-dotenv, requests, tqdm
Required-by:
```

```
~ % google-chrome --version
  google-chrome --version
  shell: /usr/bin/bash -e {0}
Google Chrome 114.0.57[3](https://github.com/ivanovmp/Webdriver-Manager-Testing/actions/runs/5358131043/jobs/9719976985#step:5:3)5.133
```

**Comment by ivanovmp** [comment]
Is there anything else I can tell you about the situation? If it's worth anything, my issue is completely reproducible, you can clone my repository https://github.com/ivanovmp/Webdriver-Manager-Testing an any moment.

**Comment by SergeyPirogov** [comment]
Fixed in 3.9.0
